packages/shared/src/components/Feed.tsx

```diff
@@ -125,6 +129,11 @@ const getStyle = (useList: boolean, spaciness: Spaciness): CSSProperties => {
   return {};
 };
 
+const PostModalMap = {
+  [PostType.Article]: PostModal,
+  [PostType.Share]: SquadPostModal,
+};
```

Comment:
Maybe just to be safe make it a Record type?

Reply:
Totally agree with that.